src/views/article.vue

```diff
@@ -28,21 +28,23 @@
                 <div class="text-box">
                     <img class="text-box-deco desktop-only" src="../assets/groups/paragraph-vector.svg" alt="">
                     <h2 class="text-box-title">{{ chapter.subtitle }}</h2>
-                    <p v-if="chapter.paragraph" class="text-box-text" v-html="getParsedText(chapter.paragraph)"></p>
-                    <!-- LIST -->
-                    <ul class="list" v-if="chapter.list">
-                        <li v-for="bulletPoint in chapter.list" v-html="getParsedText(bulletPoint)"></li>
-                    </ul>
-                    <!-- TABLE -->
-                    <table v-if="chapter.table">
-                        <tr v-if="chapter.table.header">
-                            <th v-for="(content, i) in chapter.table.header" v-html="getParsedText(content)"
-                                @click="sortTable(chapter.table, i)" class="clickable"></th>
-                        </tr>
-                        <tr v-for="row in chapter.table.rows">
-                            <td v-for="content in row" v-html="getParsedText(content)"></td>
-                        </tr>
-                    </table>
+                    <div v-for="(dynamicContent) in chapter.dynamicContents" :key="dynamicContent.index">
```

Comment:
that's the correct approach 👍 

`dynamicContent.index` doesn't make sense, since there is no index property in dynamicContent